test/e2e/apimachinery/etcd_failure.go

```diff
@@ -83,28 +83,28 @@ func etcdFailTest(f *framework.Framework, failCommand, fixCommand string) {
 	apps.TestReplicationControllerServeImageOrFail(f, "basic", framework.ServeHostnameImage)
 }
 
-// For this duration, etcd will be failed by executing a failCommand on the master.
+// For this duration, etcd will be failed by executing a failCommand on the instance.
 // If repeat is true, the failCommand will be called at a rate of once per second for
 // the failure duration. If repeat is false, failCommand will only be called once at the
 // beginning of the failure duration. After this duration, we execute a fixCommand on the
-// master and go on to assert that etcd and kubernetes components recover.
+// instance and go on to assert that etcd and kubernetes components recover.
 const etcdFailureDuration = 20 * time.Second
 
 func doEtcdFailure(failCommand, fixCommand string) {
 	ginkgo.By("failing etcd")
 
-	masterExec(failCommand)
+	instanceExec(failCommand)
 	time.Sleep(etcdFailureDuration)
-	masterExec(fixCommand)
+	instanceExec(fixCommand)
 }
 
-func masterExec(cmd string) {
+func instanceExec(cmd string) {
```

Comment:
would a better term be etcdInstanceExec?

Reply:
controlPlaneInstance